CONTRIBUTING.md

```diff
@@ -31,30 +31,30 @@ $ git clone https://github.com/<username>/command-sets
 ### 2. Create a new directory named `greetings` inside the `command-sets` repository.
 
 ```sh
-$ cd command-sets && md greetings
+$ cd command-sets && mkdir -p greetings/packages/default
 ```
 
 ### 3. Create two new files inside `greetings` to greet in Spanish & French.
 
 ```sh
-$ cd greetings && touch {hola,bonjour}.js
+$ cd greetings/packages/default && touch {hola,bonjour}.js
 ```
 
-Here each file represent a command.
+Here each file represents a command.
 
-### 4. Create a `commands.yaml` file.
+### 4. Create a `commands.yaml` file inside `greetings`
 
-This is to let commander know about our commands & their requirements.
+This is to let Commander know about our commands & their requirements.
 
 ```sh
-$ touch commands.yaml
+$ cd ../../greetings && touch commands.yaml
 ```
 
 Copy & paste the below into `commands.yaml`.
 
 ```yaml
 # We need to provide the raw resource link since commander fetches the code directly from here.
-sourceBasePath: https://raw.githubusercontent.com/nimbella/command-sets/master/greetings
+sourceBasePath: https://raw.githubusercontent.com/nimbella/command-sets/master/greetings/packages/default
 commands:
   hola:
     sourcePath: /hola.js # Path of the command relative to sourceBasePath
```

Comment:
```suggestion
    sourcePath: /packages/default/hola.js # Path of the command relative to sourceBasePath
```

Reply:
you'd also need to change `    sourcePath: /bonjour.js` if you accept this change.